pkg/asset/quota/quota.go

```diff
@@ -118,7 +119,7 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 			return summarizeFailingReport(reports)
 		}
 		summarizeReport(reports)
-	case azure.Name, baremetal.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name, vsphere.Name:
+	case azure.Name, baremetal.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name, vsphere.Name, kubevirt.Name:
```

Comment:
This is where you should check things such as that the user can list VMs in the namespace.

Reply:
Here is the quota validation, verify that user has enough resources available to create the cluster, no?
Opened a task to investigate how it should be done in kubevirt (OCPCNV-147)